Global/OSX.gitignore

```diff
@@ -1,24 +1,26 @@
-.DS_Store
-.AppleDouble
-.LSOverride
-
-# Icon must end with two \r
-Icon

-
-# Thumbnails
-._*
-
-# Files that might appear in the root of a volume
-.DocumentRevisions-V100
-.fseventsd
-.Spotlight-V100
-.TemporaryItems
-.Trashes
-.VolumeIcon.icns
-
-# Directories potentially created on remote AFP share
-.AppleDB
-.AppleDesktop
-Network Trash Folder
-Temporary Items
-.apdisk
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.temp

```

Comment:
It's generated by any Adobe, Microsoft, or other applications of the sort
that have AppData.

On Sunday, July 5, 2015, Brendan Forster <notifications@github.com> wrote:

> In Global/OSX.gitignore
> <https://github.com/github/gitignore/pull/1583#discussion_r33900165>:
>
> > +
> > +# Icon must end with two \r
> > +Icon
> > +
> > +
> > +# Thumbnails
> > +._*
> > +
> > +# Files that might appear in the root of a volume
> > +.DocumentRevisions-V100
> > +.fseventsd
> > +.Spotlight-V100
> > +.TemporaryItems
> > +.Trashes
> > +.VolumeIcon.icns
> > +.temp
>
> Do you know what tool might be responsible for generating this .temp file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/github/gitignore/pull/1583/files#r33900165>.
>


-- 


  *Max Tuchman*
------------------------------------------

------------------------------------------
Contact me:
me@mbtuchman.com
mbtuchman@maxdev.us

*DISCLAIMER:*
The information contained in this e-mail and its attachments are intended
solely for the use of the individual(s) to whom it is addressed and
contains information that is confidential. If the reader of this e-mail is
not the intended recipient, you are hereby notified that any dissemination,
distribution, or copying of this e-mail is strictly prohibited. If you have
received this e-mail in error, please immediately notify the sender and
destroy the e-mail.
Thank You.